pkg/zson/record.go

```diff
@@ -42,6 +42,22 @@ func NewRecord(d *Descriptor, ts nano.Ts, raw zval.Encoding) *Record {
 	return &Record{
 		Ts:         ts,
 		Descriptor: d,
+		Stable:     true,
+		Raw:        raw,
+	}
+}
+
+// NewTempRecord creates a record from a timestamp and a raw value
+// marked temporary so that Keep() must be called to make it permanent.
```

Comment:
"marked temporary" -> "marked non-Stable" ?

Reply:
PS: Not for this PR, but I'm interpreting the the use of "temporary" and "permanent" here (as opposed to unstable and stable) as a hint that Stable is not an ideal name. It could be flipped and renamed to Volatile for example. Later...

Reply:
Oh yeah, I like volatile

Reply:
Also NewTempRecord -> NewVolatileRecord?